romancal/assign_wcs/assign_wcs_step.py

```diff
@@ -37,7 +37,14 @@ def process(self, input):
                 reffile = self.get_reference_file(input_model, reftype)
                 reference_file_names[reftype] = reffile if reffile else ""
             log.debug(f'reference files used in assign_wcs: {reference_file_names}')
-            result = load_wcs(input_model, reference_file_names)
+            result = load_wcs(input_model.copy(), reference_file_names)
```

Comment:
Why a copy?

Reply:
Cleaned this up a leftover from testing.